build/build-image/Dockerfile

```diff
@@ -87,6 +92,19 @@ RUN wget -q https://static.rust-lang.org/rustup/archive/1.11.0/${RUST_ARCH}/rust
 RUN cargo install protobuf-codegen --vers 2.0.2
 RUN cargo install grpcio-compiler --vers 0.3.0
 
+# set up Consul.
```

Comment:
I take it we need consul locally to run e2e against local clusters?

Reply:
No we don’t, I left it in case we would need it for testing but I’m happy to remove it, WDYT ?

Reply:
I'd say let's remove it - it's an added step we have then have to maintain, and keep up to date.